search_username/catalog/views.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import render
+from InstagramAPI import InstagramAPI
```

Comment:
Please fix your imports according to PEP-8.

Reply:
row 1 or 2?I don't quite understand

Reply:
Read PEP-8 and topics related to imports.

Reply:
done

Reply:
It's not fixed according to PEP-8 yet.

Reply:
grouped